docs/source-pytorch/extensions/logging.rst

```diff
@@ -71,16 +71,14 @@ You can also pass a custom Logger to the :class:`~lightning.pytorch.trainer.trai
 
 Choose from any of the others such as MLflow, Comet, Neptune, WandB, etc.
 
-.. testcode::
-    :skipif: not _COMET_AVAILABLE
+.. code-block:: python
```

Comment:
Why not test it?

Reply:
This wasn't tested before. We don't install the logger dependencies in the docs build.

Reply:
so let;s install it

Reply:
We can look into this separately. Since this PR only changes imports, there is no need to do it here.